app/uk/gov/hmrc/tariffclassificationfrontend/forms/SearchFormData.scala

```diff
@@ -20,20 +20,22 @@ import play.api.data.Form
 import play.api.data.Forms._
 import uk.gov.hmrc.tariffclassificationfrontend.models.Search
 
-case class SearchFormData(traderName: String = "")
+case class SearchFormData(traderName: String = "", commodityCode: String = "")
```

Comment:
FYI I dont think forms can handle Options so it has to be an empty String... It gets filtered though in the Query String Binders